src/hub.js

```diff
@@ -181,6 +190,12 @@ async function enterScene(mediaStream, enterInVR, janusRoomId) {
   });
 
   if (!qsTruthy("offline")) {
+    document.body.addEventListener("connected", () => {
+      hubChannel.sendEntryEvent().then(() => {
+        store.update({ lastEnteredAt: moment().format() });
```

Comment:
`moment().format()` is in the user's current timezone. We should use UTC, no? `moment().toJSON()` does that.

Reply:
More importantly, `moment().format()` does not contain timezone information.

Reply:
Oh, nevermind. We're using moment-timezone, which _does_ return timezone information 😑

Reply:
Cool, you're right though we should store it in UTC